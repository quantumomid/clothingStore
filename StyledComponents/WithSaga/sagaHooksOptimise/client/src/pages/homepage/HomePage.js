import { Profiler } from "react";
import Directory from "../../components/directory/Directory";
import { HomePageContainer } from "./homepageStyles";

export default function HomePage(){
    return(
        <HomePageContainer>
            <Profiler id="Directory" onRender={(id, phase, actualDuration) => console.log({ id, phase, actualDuration })}>
                <Directory />
            </Profiler>
        </HomePageContainer>
    )
}