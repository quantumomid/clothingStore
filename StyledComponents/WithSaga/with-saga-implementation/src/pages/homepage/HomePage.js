import Directory from "../../components/directory/Directory";
import { HomePageContainer } from "./homepageStyles";

export default function HomePage(){
    return(
        <HomePageContainer>
            <Directory />
        </HomePageContainer>
    )
}