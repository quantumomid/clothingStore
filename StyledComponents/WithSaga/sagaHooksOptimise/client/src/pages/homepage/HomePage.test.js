import React from "react";
import { shallow } from "enzyme";
import HomePage from "./HomePage";

describe("HomePage Component", () => {
    let wrapper;
  
    beforeEach(() => {
        wrapper = shallow(<HomePage />);
    });
  
    it("HomePage Component should render", () => {
        expect(wrapper).toMatchSnapshot();
    })
})