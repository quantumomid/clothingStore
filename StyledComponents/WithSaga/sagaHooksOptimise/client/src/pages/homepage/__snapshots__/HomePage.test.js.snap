// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Component HomePage Component should render 1`] = `ShallowWrapper {}`;
